README.md

```diff
@@ -65,6 +66,7 @@ Epoxy has a modular architecture so you only have to include what you need for y
 | [`EpoxyNavigationController`](#EpoxyNavigationController) | Declarative API for driving the navigation stack of a [`UINavigationController`](https://developer.apple.com/documentation/uikit/uinavigationcontroller) |
 | [`EpoxyPresentations`](#EpoxyPresentations) | Declarative API for driving the modal presentations of a [`UIViewController`](https://developer.apple.com/documentation/uikit/uiviewcontroller) |
 | [`EpoxyBars`](#EpoxyBars) | Declarative API for adding fixed top/bottom bar stacks to a [`UIViewController`](https://developer.apple.com/documentation/uikit/uiviewcontroller) |
+| [`EpoxyLayoutGroups`](#EpoxyLayoutGroups) | Declarative API for driving the subviews of a UIView |
```

Comment:
Same suggestion here: It might be nice to include some details about how this supports nesting/is composable since it allows creating most layouts. Also the flattening logic is novel but not sure if you want to mention that.